components/clarity-vscode/README.md

```diff
@@ -28,11 +65,12 @@ From the `./components/clarity-vscode`, run `npm install` to install the depende
 ### Structure
 
 The LSP has two main parts: the client and the server.
-This two part will run differents environments:
-- VSCode Web: in a WebWorkers
+This two part will run different environments:
+- VSCode Web:  for Web Workers' environment
+- VSCode Desktop: for Node.js environment```
```

Comment:
Something is odd here, formatting-wise.

Reply:
Indeed, something went wrong with the github suggestion I'll clean it